crates/cxx-qt-gen/src/generator/cpp/signal.rs

```diff
@@ -521,7 +521,7 @@ mod tests {
         let mut type_names = TypeNames::default();
         type_names.mock_insert("ObjRust", None, None, None);
         let qobject_name = type_names.lookup(&signal.qobject_ident).unwrap();
-        let generated = generate_cpp_signal(&signal, &qobject_name, &type_names).unwrap();
+        let generated = generate_cpp_signal(&signal, qobject_name, &type_names).unwrap();
```

Comment:
where is this change coming from as the signature hasn't changed ? Seems like a clippy style fix ?

Reply:
yeah, was "immediately dereferencing" warning, so assuming qobject_name was already a reference, so function got &&T and just instantly coerced to &T, so it wasn't needed

Reply:
Even running with `--all-targets` doesn't help fun :-)